From 87eab684b1e24ce3cfd88625120fa4cb623410c2 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 22 Jul 2014 14:38:37 -0400
Subject: [virt] kvm/x86: Fix debug typo error in lapic

Message-id: <1406040016-3289-123-git-send-email-pbonzini@redhat.com>
Patchwork-id: 86201
O-Subject: [RHEL7 PATCH v2 122/221] KVM: x86: Fix debug typo error in lapic
Bugzilla: 1116936
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Chen Fan <chen.fan.fnst@cn.fujitsu.com>

fix the 'vcpi' typos when apic_debug is enabled.

Signed-off-by: Chen Fan <chen.fan.fnst@cn.fujitsu.com>
Signed-off-by: Marcelo Tosatti <mtosatti@redhat.com>
(cherry picked from commit 96893977b8f732493815e7a2b552c37e1bb967e5)
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kvm/lapic.c b/arch/x86/kvm/lapic.c
index d86ff15..9736529 100644
--- a/arch/x86/kvm/lapic.c
+++ b/arch/x86/kvm/lapic.c
@@ -432,7 +432,7 @@ static bool pv_eoi_get_pending(struct kvm_vcpu *vcpu)
  u8 val;
  if (pv_eoi_get_user(vcpu, &val) < 0)
   apic_debug("Can't read EOI MSR value: 0x%llx\n",
-      (unsigned long long)vcpi->arch.pv_eoi.msr_val);
+      (unsigned long long)vcpu->arch.pv_eoi.msr_val);
  return val & 0x1;
 }
 
@@ -440,7 +440,7 @@ static void pv_eoi_set_pending(struct kvm_vcpu *vcpu)
 {
  if (pv_eoi_put_user(vcpu, KVM_PV_EOI_ENABLED) < 0) {
   apic_debug("Can't set EOI MSR value: 0x%llx\n",
-      (unsigned long long)vcpi->arch.pv_eoi.msr_val);
+      (unsigned long long)vcpu->arch.pv_eoi.msr_val);
   return;
  }
  __set_bit(KVM_APIC_PV_EOI_PENDING, &vcpu->arch.apic_attention);
@@ -450,7 +450,7 @@ static void pv_eoi_clr_pending(struct kvm_vcpu *vcpu)
 {
  if (pv_eoi_put_user(vcpu, KVM_PV_EOI_DISABLED) < 0) {
   apic_debug("Can't clear EOI MSR value: 0x%llx\n",
-      (unsigned long long)vcpi->arch.pv_eoi.msr_val);
+      (unsigned long long)vcpu->arch.pv_eoi.msr_val);
   return;
  }
  __clear_bit(KVM_APIC_PV_EOI_PENDING, &vcpu->arch.apic_attention);
-- 
1.7.1